.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "plugins": ["@babel/transform-runtime"],
```

Comment:
Can this be left out? lib-size isn't as important for webpack plugins. I'm seeing issues with the runtime not getting hoisted correctly in node_modules.

Reply:
Actually, I don't think it's a node_module structure issue. I think the latest babel beta changed the runtime api... this project probably needs to be recompiled and published... preferably without the runtime transform.

Reply:
I've bumped all the dependencies in [this commit](https://github.com/GoogleChromeLabs/preload-webpack-plugin/pull/72/commits/d80459ad5941e0535c68dd27708590ceafee34ab), including babel-related ones to `7.0.0-beta.56`. Does that help at all?

That being said, I'm not sure why shipping a project like this that relied on `@babel/runtime` as a dependency would be an issue. Is there any official guidance from the babel team about that?

Reply:
Okay, fair enough. If it helps, I've removed the `transform-runtime` in the [latest commit](https://github.com/GoogleChromeLabs/preload-webpack-plugin/pull/72/commits/e58a8379bd9d4c3528ee357c1c77489782d5793f).